docs/broken_links.py

```diff
@@ -0,0 +1,58 @@
+import re
+import os
+import argparse
+
+from colorama import Fore, Back, Style
+
+files = list()
+os.chdir("source")
+path = os.getcwd()
+broken_count = 0
+clean_count = 0
+
+parser = argparse.ArgumentParser(description='Check broken links')
+parser.add_argument('--verbose', default=False, help="Verbosity level (1: Show all links)")
+
+args = parser.parse_args()
+
+def shortpath(path):
+    return os.path.relpath(path)
+
+def check_broken_links(file):
+    global broken_count
+    global clean_count
```

Comment:
Meaningless to use `global` here, when these things can easily be returned from the function.